tsl/src/compression/create.h

```diff
@@ -18,6 +18,8 @@
 
 bool tsl_process_compress_table(AlterTableCmd *cmd, Hypertable *ht,
 								WithClauseResult *with_clause_options);
+void tsl_process_compress_table_add_column(int32 orig_htid, int32 compress_htid,
+										   ColumnDef *orig_def);
```

Comment:
The name `orig_def` suggests that it will not be modified and it is not in the implementation, thus:
```suggestion
void tsl_process_compress_table_add_column(int32 orig_htid, int32 compress_htid,
										   const ColumnDef *orig_def);
```